enhancements/OSAC-1436-caas-networking/design.md

```
@@ -99,15 +99,15 @@ These steps are identical to VMaaS/BMaaS — the networking API is uniform.
 
 1. **Create VirtualNetwork:**
    ```bash
-   osac create virtualnetwork --region moc-region-1 --cidr 10.0.0.0/16 --name my-net
+   osac create virtualnetwork --network-class moc-bm-1 --cidr 10.0.0.0/16 --name my-net
    ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)